solution-base/mongodb/charts/mongodb-sharded/values.yaml

```diff
@@ -229,26 +235,25 @@ common:
     annotations: {}
     ## @param common.serviceAccount.automountServiceAccountToken Automount service account token for the server service account
     ##
-    automountServiceAccountToken: true
-
+    automountServiceAccountToken: false
```

Comment:
this was changed to "stop" auto-mounting service account token : what is the purpose / is there an impact?

esp. considering we do not actually create a `ServiceAccount` (serviceAccount.create == false), so I would think not?